src/inference/src/dev/core_impl_ie.cpp

```diff
@@ -0,0 +1,385 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <memory>
+
+#include "any_copy.hpp"
+#include "compilation_context.hpp"
+#include "core_impl.hpp"
+#include "cpp_interfaces/interface/ie_internal_plugin_config.hpp"
+#include "cpp_interfaces/interface/ie_iplugin_internal.hpp"
+#include "dev/converter_utils.hpp"
+#include "ie_network_reader.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/pass/constant_folding.hpp"
+#include "openvino/itt.hpp"
+
+bool ov::CoreImpl::isNewAPI() const {
+    return is_new_api();
+}
+
+ov::SoPtr<InferenceEngine::IExecutableNetworkInternal> ov::CoreImpl::LoadNetworkImpl(
+    const InferenceEngine::CNNNetwork& network,
+    ov::Plugin& plugin,
+    const std::map<std::string, std::string>& parsedConfig,
+    const InferenceEngine::RemoteContext::Ptr& context,
+    const CacheContent& cacheContent,
+    bool forceDisableCache) {
+    OV_ITT_SCOPED_TASK(ov::itt::domains::IE, "CoreImpl::compile_model_impl");
+    ov::SoPtr<InferenceEngine::IExecutableNetworkInternal> execNetwork;
+    auto wrapper = std::dynamic_pointer_cast<InferenceEngine::IPluginWrapper>(plugin.m_ptr);
+    OPENVINO_ASSERT(wrapper);
+    auto old_plugin = wrapper->get_plugin();
+    execNetwork = {context ? old_plugin->LoadNetwork(network, parsedConfig, context)
+                           : old_plugin->LoadNetwork(network, parsedConfig),
```

Comment:
why do we need explicitly call old_plugin API? We have IPluginWrapper which should bypass everything to new plugin api

Reply:
It is needed for IR v7 only. I cannot convert IR v7 to ov::Model